### ISSUE TITLE
Unible to set local anime folder

### ISSUE THREAD

**Issue by juniornff** [z411/trackma#673]
I just installed Tackma in my laptop (Zorin OS 15.3 i686) and I launch it with -curses (because the other options show me errors, I'll comment on that in another issue)

I read that in config.json you can set the directory to search for videos. I have placed both the general and anime-specific folders and even so I can't get it to play or even open the localfolder in trackma. Is there a correct way to do it?

I also read elsewhere that it might have something to do with giving permissions to those specific folders. Any help with that if it is the case?

I leave my config file for any reference. Also i want to uso VLC as player, Did I put the parameter correctly?

[config (copia 1).zip](https://github.com/z411/trackma/files/10763133/config.copia.1.zip)

**Comment by z411** [comment]
Seems to be fine. Can you launch the Trackma interface with debugging information enabled, try to play an episode and post the output? (`trackma -d`)

**Comment by juniornff** [comment]
This is the output
```
Engine warning: The specified media directory /home/familia/Videos/Animes doesn't exist!
Engine warning: The specified media directory /home/familia/Videos/Animes/Blue Lock doesn't exist!
Engine warning: The specified media directory /home/familia/Videos/Animes/Eizouken ni wa Te wo Dasu na! doesn't exist!
```
Clarify that those folders are created

**Comment by juniornff** [comment]
Quick update, To test copy the folders to another location, from
`/home/familia/Videos/Animes`
to
`/home/familia/Animes`
And with that Trackma was able to recognize the chapters and reproduce them correctly. Apparently it is a permissions issue with the folders auto generated by the OS

I imagine that it is not a problem of trackma but of the OS permissions, but it would be good to confirm if there is a solution to that, because I imagine that there will be people who are not willing to try so much trial and error.

In short, the solution is to place the anime directory in the user's root directory, avoiding autogenerated folders.
